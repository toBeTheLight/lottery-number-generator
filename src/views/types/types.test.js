import React from 'react';
import ReactDom from 'react-dom';
import Types from './types'
import { mount } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';
import { HashRouter as Router} from 'react-router-dom'

describe('seven-star', () => {
  it('renders without crashing', () => {
    const wrapper = mount(
      <Router>
        <Types />
      </Router>);
    expect(renderToJson(wrapper)).toMatchSnapshot();
  })
})