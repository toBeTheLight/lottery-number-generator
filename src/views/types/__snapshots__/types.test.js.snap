// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`seven-star renders without crashing 1`] = `null`;
